src/site/xdoc/index.xml

```diff
@@ -40,6 +40,8 @@
           Java POJOs
           (Plain Old Java Objects) to database records.
         </p>
+        <p>If you are completely new to database access in Java,<a href="https://www.marcobehler.com/guides/a-guide-to-accessing-databases-in-java">https://www.marcobehler.com/guides/a-guide-to-accessing-databases-in-java</a> contains a good overview of the various parts, pieces and options and shows you where exactly MyBatis fits in.
```

Comment:
Is this a good place for advertising？

Reply:
I don't know about connotating it as advertising (for what exactly?). 

It is outreach for an article that helped quite a few people so far get a good picture of the Java DB landscape - that's why I think places like the Hibernate or jOOQ documentation are also already linking to it.

Reply:
> Is this a good place for advertising？

Although I also think it is not appropriate to add a personal website that is not widely recognized in MyBatis document, I think there should be hints for beginners in the document. #2480